design-system/materials/internals/definition.yaml

```diff
@@ -62,18 +62,18 @@ choiceGroupsByTheme:
         color-neutral-95: 'hsl(0, 0%, 95%)'
         color-neutral-98: 'hsl(0, 0%, 98%)'
         color-info: '#078cdf'
-        color-info-40: 'hsl(203.05555555555554, 93.9130434783%, 40%)'
+        color-info-40: 'hsl(203.05555555555554, 93.9130434783%, 35%)'
```

Comment:
Maybe we should be creating a new token here (and the same for warning and error tokens) since the name is following some implicit convention and this would change not only `Stamp` component component but any other which uses this token.
In `ui-kit` there's only this component using this token, but it's an exported one so any other external consumer would have this color change in whatever context they're using it.

Reply:
I'm holding off on this PR until the Color and Typography stories get completed. @valoriecarli  is working on the color story and can get the colors defined as we will need them for stamps.

So it sounds like you are asking that we go back to adding new <color>-35 tokens for all the stamp colors?

Reply:
This PR no longer changes colors of the stamps as specified in the updated Figma.